docs/cache-and-resume.md

```diff
@@ -22,10 +22,10 @@ The task hash is computed from the following metadata:
 - Task {ref}`environment modules <process-module>` (if applicable)
 - Task {ref}`Conda environment <process-conda>` (if applicable)
 - Task {ref}`Spack environment <process-spack>` and {ref}`CPU architecture <process-arch>` (if applicable)
-- Task {ref}`process-ext` directive (if applicable)
 - Task {ref}`inputs <process-input>`
 - Task {ref}`script <process-script>`
 - Any global variables referenced in the task script
+- Any task {ref}`process-ext` properties referenced in the task script
```

Comment:
If I remember correctly there were some exceptions

Reply:
As far as I can tell it is just the ext variables that are referenced in the script: https://github.com/nextflow-io/nextflow/blob/1c2ef7b837903b6d331b4ba360e9e10f7cac2da6/modules/nextflow/src/main/groovy/nextflow/processor/TaskProcessor.groovy#L2319-L2330

Reply:
ok, the exception is all `.ext.*` 😄 https://github.com/nextflow-io/nextflow/pull/4339